ci/requirements-3.4.pip

```diff
@@ -1,3 +1,5 @@
-httplib2
-google-api-python-client
-oauth2client
+httplib2>=0.9.1
```

Comment:
Can we remove `httplib2` from `setup.py` altogether and use `from googleapiclient.http import build_http` in the code?


```
http = httplib2.Http()
```
becomes
```
http = build_http()
```